janus_server/src/datastore.rs

```diff
@@ -76,7 +76,8 @@ pub struct Transaction<'a> {
 impl Transaction<'_> {
     // TODO(brandon): implement basic getters/putters for all types
 
-    #[cfg(test)]
+    // This is pub to be used in integration tests
+    #[doc(hidden)]
```

Comment:
This is bad, as we discussed in another PR. I'm happy to hold off on this change until we do #18, and then I can rebase on that.

Reply:
ah well. this will eventually be expanded into a real `pub` method, anyway...